assets/samples/edgeconnect/provisioner.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: dynatrace.com/v1alpha2
+kind: EdgeConnect
+metadata:
+  name: edge-connect-sample
+  namespace: dynatrace
+spec:
+  apiServer: "https://example.dev.apps.dynatracelabs.com"
+  replicas: 1
+
+  oauth:
+    # OAuth client has to be created in Dynatrace account management
```

Comment:
same here, I'm assuming there's a guide somewhere for this

Reply:
note sure but found something similar https://docs.dynatrace.com/docs/manage/configuration-as-code/terraform/guides/create-oauth-client

Reply:
Yeah, there is no "general" description, but there are a bunch of paragraphs that can be helpful.

Reply:
8bb546cca5e0c5f0c601d9eb380c4c95d6e0349e

Reply:
1978807a